experiment/nursery/main.go

```diff
@@ -142,3 +176,42 @@ func cacheHandler(cache *diskcache.Cache) http.Handler {
 		}
 	})
 }
+
+func monitorDiskAndEvict(cache *diskcache.Cache, interval time.Duration, minBlocksFree, minFilesFree float64) {
```

Comment:
Consider moving this to the `diskcache` package and making it a method of `Cache`.

Reply:
SGTM